src/main/java/max/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,48 @@
+package max.commands;
+import max.commands.*;
+import max.exception.MaxException;
+import max.parser.Parser;
+import max.storage.Storage;
+import max.tasks.TaskList;
+import max.tasks.Task;
+import max.ui.Ui;
+
+/**
+ * Represents list command.
+ */
+public class UnmarkCommand extends Command {
+    public static final String COMMAND_WORD = "unmark";
```

Comment:
Would it be possible to make the variable private?

Reply:
Thanks for pointing this out! Perhaps I will implement a public `getCommandWord()` method instead. Do you think this would be better?

Reply:
Yup, I think this would be better. Though, I was thinking if you could use enums?